subset/switches/src/main/java/switchtest/SwitchTest.java

```diff
@@ -0,0 +1,156 @@
+package switchtest;
+
+import grpc.InterfaceResponse;
+import grpc.LinkStatus;
+import grpc.POENegotiation;
+import grpc.POEStatus;
+import grpc.POESupport;
+import grpc.PowerResponse;
+import grpc.SwitchInfo;
+import grpc.USIServiceGrpc;
+import io.grpc.Channel;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.concurrent.TimeUnit;
+
+public class SwitchTest {
+
+  private final USIServiceGrpc.USIServiceBlockingStub blockingStub;
+  protected String reportFilename = "tmp/report.txt";
+  protected boolean debug;
+  protected boolean deviceConfigPoeEnabled;
+  protected int rpcTimeoutSec;
+
+  /**
+   * Generic switch test.
+   * @param channel GRPC channel
+   * @param rpcTimeoutSec Timeout in seconds for rpc calls
+   * @param deviceConfigPoeEnabled poe config from module_config
+   * @param debug print debug output
+   */
+  public SwitchTest(Channel channel, int rpcTimeoutSec, boolean deviceConfigPoeEnabled,
```

Comment:
nb: something is still wrong with your settings or the linter. This should just be hanging 4 indent (as per Google styleguide).  Not a big deal so don't need to fix it but just be aware.

Reply:
This passes the linter on my local machine and on stickler. It can only mean the checkstyle file doesn't include this check.